pkg/restic/backupper.go

```diff
@@ -150,13 +151,11 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 		}
 
 		volumeBackup := newPodVolumeBackup(backup, pod, volumeName, repo.Spec.ResticIdentifier)
-
-		if err := errorOnly(b.repoManager.veleroClient.VeleroV1().PodVolumeBackups(volumeBackup.Namespace).Create(volumeBackup)); err != nil {
+		volumeSnapshots[volumeName] = volumeBackup
```

Comment:
it doesn't look like we really need the `volumeSnapshots` map anymore - the only thing I see us using it for is to know the number of PVB's that we created and are waiting for. So I'd just keep an `int` count here instead, and use that on https://github.com/heptio/velero/pull/1577/files#diff-d86635f0e9c2aff5dfda0f868fd8062fR162